opentelemetry-api/tests/metrics/test_metrics.py

```diff
@@ -0,0 +1,86 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+
+from opentelemetry.metrics import (
+    Instrument,
+    Synchronous,
```

Comment:
The name `Synchronous/Asynchronous` might be too general?

Reply:
How about `SynchronousInstrument`?

Reply:
`Synchronous` and `Asynchronous` both inherit from `Instrument`, so they can be evaluated with `isinstance`, so the user knows.